tests/test_20_use_case_topology.py

```diff
@@ -515,15 +515,126 @@ def test_070_port_uni_up_no_l2vpn_associated(self):
+        l2vpn_response = response.json()
+        l2vpn_response = l2vpn_response.get(l2vpn_id)
+        assert l2vpn_response.get("status") == "error", l2vpn_response
+
+        # Expanding vlan
+
+        payload = {
+            "sdx_vlan_range": [[100,2000]]
+        }
+        response = requests.post(f"{api_url_ampath}/interfaces/{interfaces_id}/metadata", json=payload)
+        assert response.status_code == 201, response.text
+
+        # Force to send the topology to the SDX-LC
+        sdx_api = KYTOS_SDX_API % 'ampath'
+        response = requests.post(f"{sdx_api}/topology/2.0.0")
+        assert response.status_code == 200, response.text
+
+        time.sleep(15)
+
+        response = requests.get(API_URL)
+        assert response.status_code == 200, response.text
+        l2vpn_response = response.json()
+        l2vpn_response = l2vpn_response.get(l2vpn_id)
+        assert l2vpn_response.get("status") == "up", l2vpn_response
+
+    @pytest.mark.xfail(reason="The L2VPN with VLAN 1040 remains up even after modifying the VLAN range to [100–200]")
+    def test_104_expanding_vlan_nni_no_alternate_path(self):
         """
-        Use Case 11: OXPO sends a topology update with a service no longer being supported on a certain Port
+        Use Case 10: OXPO sends a topology update with a changed VLAN range is for any of the services supported.
         """
+
+        l2vpn_data = self.create_new_l2vpn(vlan='1040')
+        l2vpn_id = l2vpn_data['id']
+
+        interfaces_id = 'cc:00:00:00:00:00:00:06:41'
+        interface_name = 'Tenet01-eth41'
+        tenet_api = KYTOS_API % 'tenet'
+        api_url_tenet = f'{tenet_api}/topology/v3'
+        payload = {"sdx_vlan_range": [[100,200]]}
+        response = requests.post(f"{api_url_tenet}/interfaces/{interfaces_id}/metadata", json=payload)
+        assert response.status_code == 201, response.text
+
+        # Avoid alternative path
+        interfaces_id_2 = 'cc:00:00:00:00:00:00:06:1'
+        response = requests.post(f"{api_url_tenet}/interfaces/{interfaces_id_2}/metadata", json=payload)
+        assert response.status_code == 201, response.text
+
+        # Force to send the topology to the SDX-LC
+        sdx_api = KYTOS_SDX_API % 'tenet'
+        response = requests.post(f"{sdx_api}/topology/2.0.0")
+        assert response.status_code == 200, response.text
+
+        time.sleep(15)
+
+        response = requests.get(API_URL_TOPO)
+        data = response.json()
+        for node in data['nodes']: 
+            for port in node['ports']:
+                if port['name'] == interface_name:
+                    assert port['services']['l2vpn_ptp']['vlan_range'] == [[100,200]], port
+
+        response = requests.get(API_URL)
+        assert response.status_code == 200, response.text
+        l2vpn_response = response.json()
+        l2vpn_response = l2vpn_response.get(l2vpn_id)
+        assert l2vpn_response.get("status") == "error", l2vpn_response
```

Comment:
I didnt understand why the L2VPN would be in an error state here. It will find alternative paths.

Reply:
That's right, the idea was that by removing alternative paths the state should go to error, but by removing the previous code now the state should remain in up, just as you mentioned.